Menu/Menu.js

```diff
@@ -1,11 +1,39 @@
 //Using jQuery, select the menu element and the menu-button element
-const menu;
-const menuButton;
+const menu = $('.menu');
+const menuButton = $('.menu-button');
 
 //Create a callback that will be passed to the menuButton click event listener
 //Inside this callback a class should be either added or removed from menu
 const toggleMenu = () => {
-
+    $('.menu').toggleClass('menu--open')
```

Comment:
We should reference an element passed into it or a query passed into it or both to make this truely reusable anywhere.